test/json-pipe.test.js

```diff
@@ -536,7 +536,7 @@ describe('JSON Pipe Test', () => {
     });
     const resp = await jsonPipe(state, new PipelineRequest('https://json-filter.com/?limit=5'));
     assert.strictEqual(resp.status, 502);
-    assert.strictEqual(resp.headers.get('x-error'), 'failed to parse json: Unexpected token \'h\', "this is no json" is not valid JSON');
+    assert.strictEqual(resp.headers.get('x-error'), 'failed to parse json: Unexpected token h in JSON at position 1');
```

Comment:
this (and 2 other) JSON error comparisons were fixed in branch `5.x` but not in main